glue_jupyter/bqplot/common/tools.py

```diff
@@ -582,7 +603,10 @@ def press(self, x, y):
             if layer.visible and isinstance(subset_state, RoiSubsetState):
                 roi = subset_state.roi
                 if roi.defined() and roi.contains(x, y):
-                    if isinstance(roi, (EllipticalROI, CircularROI)):
+                    if isinstance(roi, EllipticalROI):
+                        self._active_tool = BqplotEllipseMode(
+                            self.viewer, roi=roi, finalize_callback=self.release)
+                    elif isinstance(roi, CircularROI):
```

Comment:
Is the issue related to the code here in that in the case where this was created from the true circle mode, we should actually be instantiating BqplotTrueCircleMode? I don't understand why I don't see the issue though.